<?xml version="1.0"?>

<launch>

  <include file="$(find gazebo_ros)/launch/empty_world.launch"/>

  <!--Spawn simulated quadrotor uav -->
  <group ns="uav1">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
    <arg name="name" value="uav1"/>
    <arg name="tf_prefix" value="uav1" />
  </include>
</group>

<group ns="uav2">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
    <arg name="name" value="uav2"/>
    <arg name="tf_prefix" value="uav2" />
    <arg name="x" value="-2"/>
  </include>
</group>

<group ns="uav3">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
    <arg name="name" value="uav3"/>
    <arg name="tf_prefix" value="uav3" />
    <arg name="x" value="2"/>
  </include>
</group>

<group ns="uav4">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
    <arg name="name" value="uav4"/>
    <arg name="tf_prefix" value="uav4" />
    <arg name="y" value="2"/>
  </include>
</group>

<group ns="uav5">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
    <arg name="name" value="uav5"/>
    <arg name="tf_prefix" value="uav5" />
    <arg name="y" value="-2"/>
  </include>
</group>

</launch>
